SygicMapsKit/Classes/BrowseMap/SYMKBrowserMapView.swift

```diff
@@ -0,0 +1,53 @@
+import UIKit
+import SygicMaps
+import SygicUIKit
+
+
+class SYMKBrowserMapView: UIView {
+    
+    var mapView: SYMapView?
+    let compass = SYUICompass()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupUI()
+    }
+    
+    convenience init() {
+        self.init(frame: CGRect.zero)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Public Methods
+    
+    public func setupMapView() {
+        mapView = SYMapView()
+        guard let mapView = mapView else { return }
+        mapView.frame = bounds
+        mapView.geoCenter = SYGeoCoordinate(latitude: 48.147, longitude: 17.101878)! //todo: set some default
```

Comment:
ked chces nastavit kameru hned pri vytvarani mapview, supni to ako parametre do `mapView = SYMapView(...)`

Reply:
oks moze byt, myslel som to tak ze tam supneme nejaky default geocenter s nejakym vyssim zoom levelom a potom ked sa bude riesit gps lokacia spolu s lock buttonom, tak sa to potom nastavi na gps polohu

Reply:
je to len view, takze tu by mali byt nastavenia kamery ako argumenty metody. controller rozhodne co tam posle, zatial kludne ta konkretna pozicia.